src/Google.Cloud.Functions.Hosting/Logging/SimpleConsoleLogger.cs

```diff
@@ -37,12 +42,54 @@ protected override void LogImpl<TState>(LogLevel logLevel, EventId eventId, TSta
                 _ => throw new ArgumentOutOfRangeException(nameof(logLevel))
             };
 
-            Console.WriteLine($"{DateTime.UtcNow:yyyy-MM-dd'T'HH:mm:ss.fff'Z'} [{Category}] [{briefLevel}] {formattedMessage}");
-            // Note: it's not ideal to break out of the "one line per log entry" approach here, but there's no particularly
-            // nice way of getting all the relevant information otherwise.
+            StringBuilder scopeBuilder = new StringBuilder();
+            ScopeProvider.ForEachScope(AppendScope, scopeBuilder);
+
+            _console.WriteLine(Invariant($"{DateTime.UtcNow:yyyy-MM-dd'T'HH:mm:ss.fff'Z'} [{Category}] [{briefLevel}] {formattedMessage}"));
+            // Note: it's not ideal to break out of the "one line per log entry" approach for either scopes or exceptions,
+            // but there's no particularly nice way of getting all the relevant information otherwise.
+            if (scopeBuilder.Length != 0)
+            {
+                _console.WriteLine($"Scopes: [{scopeBuilder}]");
```

Comment:
I would have expected the scopes to go first I think. But I understand that this makes sense. Maybe add some visual dependency sign like "|__" (ugly, I know)

Reply:
Hmmm... I think the lack of a timestamp makes it clear enough, personally - basically the log entry is everything from one timestamp to the next one. It's fairly "busy" already, but I think it's reasonable clear as it is.